app/controllers/api/v1/sol_prices_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SolPricesController < BaseController
+      def index
+        from = (index_params[:from] || 30.days.ago).to_datetime
+        to = (index_params[:to] || from + 30.days).to_datetime
+
+        exchange_filter = \
+          SolPrice::EXCHANGES.include?(index_params[:exchange]) ? index_params[:exchange] : SolPrice::EXCHANGES
```

Comment:
There's no point in using one-line condition statement if you're going to break this line anyway. Just use normal `if` block.

Reply:
Is this condition meant to assign default exchange? Shouldn't it be `SolPrice::EXCHANGES[0]` then?

Reply:
In my opinion it should return results from both exchanges by default.